src/main/java/com/nike/cerberus/server/config/guice/CmsGuiceModule.java

```diff
@@ -345,8 +346,10 @@ public SslContext sslContext(@Named("cms.ssl.protocolsEnabled") String protocols
                     .build();
         } else {
             logger.info("initializing SslContext using certificate from S3");
-            InputStream certificate = IOUtils.toInputStream(cmsEnvPropertiesLoader.getCertificate(), Charset.defaultCharset());
-            InputStream privateKey = IOUtils.toInputStream(cmsEnvPropertiesLoader.getPrivateKey(), Charset.defaultCharset());
+            String certificateName = injector.getInstance(Key.get(String.class, Names.named("cms.ssl.certificateName")));
```

Comment:
Why not just inject this as a parameter?

Reply:
because it wont be present for local envs